[Outlook] (manifest) non-support of activation rules in JSON manifest (#3680)
OfficeDev/office-js-docs-pr: docs/outlook/privacy-and-security.md

- Developers can request the **restricted** permission if the Outlook add-in activates on a specific type of Outlook item (appointment or message), or on specific extracted entities (phone number, address, URL) being present in the item's subject or body. For example, the following rule activates the Outlook add-in if one or more of three entities - phone number, postal address, or URL - are found in the subject or body of the current message.

> [!NOTE]

> > Activation rules as seen in this example are not supported in add-ins that use the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).

Reviewer: ```suggestion
> Activation rules, as seen in this example, aren't supported in add-ins that use the [Teams manifest for Office Add-ins (preview)](../develop/json-manifest-overview.md).
```
